feature-utils/poly-look/test/react-components/buttons/iconButton.test.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+
+import { IconButton } from "../../../src/react-components";
+
+it("Creates a basic button", () => {
```

Comment:
Nit: You're not really following RSpec style here, which would be like:

```
describe("IconButton", () => {
    it("Gets rendered correctly", () => {
        ...
    });
});
```

So you could just use `test` instead of `it`, since we're using Jest, for XUnit style:

```
test("Create a basic button", () => {
...
});
```

Reply:
I know, it was more for consistency with surrounding code. But now I'm very tempted to change it. I don't care about `it` or `test` but the absence of `describe` is bothering me 😅

Reply:
I see, funny :D Well, I think this is one of those things we can refactor as we touch it, at the price of consistency.

Reply:
There's really no consistency regarding test styles. Some use test, some use describe, some use it directly. I'm good with all of them as long as there are tests.

Reply:
Done (changed for both button test files).